utils/certs/gen_certificates.sh

```diff
@@ -167,21 +172,40 @@ function generate_server_cert () {
     ${CERTS}/server.crt"
 }
 
+function populate_clients_dir () {
+    cp "${CERTS}/admin.crt"       "${CLIENTS}/admin.crt"
+    chown daos_server.daos_server "${CLIENTS}/admin.crt"
+    chmod 640                     "${CLIENTS}/admin.crt"
+    cp "${CERTS}/agent.crt"       "${CLIENTS}/agent.crt"
+    chown daos_server.daos_server "${CLIENTS}/agetn.crt"
+    chmod 640                     "${CLIENTS}/agent.crt"
+
+    echo "Authorized Clients Certificate Files on DAOS Servers:
+    ${CLIENTS}/agent.crt
+    ${CLIENTS}/admin.crt"
+}
+
 function cleanup () {
-    rm -f "${CERTS}/*pem"
+    rm -f "${CERTS}/*.pem"
     rm -f "${CA_HOME}/agent.csr"
     rm -f "${CA_HOME}/admin.csr"
     rm -f "${CA_HOME}/server.csr"
-    rm -f "${CA_HOME}/ca.cnf"
+    rm -f "${CA_HOME}/ca.cnf" # TODO: keep for future use?
```

Comment:
Good question. My instinct would be to remove it since we regenerate it anyway.

Reply:
ok - todo removed